components/constellation/constellation.rs

```diff
@@ -223,15 +224,15 @@ struct WebrenderWGPU {
     wgpu_image_map: Arc<Mutex<HashMap<u64, webgpu::PresentationData>>>,
 }
 
-/// Servo supports tabs (referred to as browsers), so `Constellation` needs to
-/// store browser specific data for bookkeeping.
-struct Browser {
-    /// The currently focused browsing context in this browser for key events.
+/// Servo supports multiple top-level browsing contexts or “webviews”, so `Constellation` needs to
+/// store webview-specific data for bookkeeping.
+struct Webview {
```

Comment:
I think the consensus might be WebView (two words, but together) based on the [Wikipedia article](https://en.wikipedia.org/wiki/WebView) (with does also use "Webview", but more often "WebView") and just consistency with WebKit. If it's not too much of a hassle, could you change it to the camel case version?

```suggestion
struct WebView {
```